SNT/Utils.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Torch.SNT
+{
+    internal class Utils
+    {
+        public static int GetTotalLength(ReadOnlySpan<int> dimensions, int startIndex = 0)
```

Comment:
Also, couldn't this be a property on the Tensor object?

Reply:
There is a Length property in Tensor<T> (which I will map into a call to Torch nElement). But this is different because it is used before the tensor is created.